src/couch_httpd_auth.erl

```diff
@@ -240,13 +242,21 @@ cookie_authentication_handler(#httpd{mochi_req=MochiReq}=Req, AuthModule) ->
         end
     end.
 
+cookie_auth_should_issue_new_cookie(MochiReq, TimeLeft, Timeout) ->
```

Comment:
This is good name for test function, but not like this one. `maybe_issue_new_cookie` ?